Building DAG of jobs...
Singularity image location: /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg
Using shell: /bin/bash
Provided cores: 4
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	add_uns_to_h5ad
	1	all
	1	run_analysis
	3
Resources before job selection: {'_cores': 4, '_nodes': 9223372036854775807}
Ready jobs (1):
	run_analysis
Selected jobs (1):
	run_analysis
Resources after job selection: {'_cores': 3, '_nodes': 9223372036854775806}

[Fri Nov 13 14:07:35 2020]
rule run_analysis:
    input: /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/SRS4137616/counts.RData, /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/SRS4137620/counts.RData
    output: GSE123587.h5ad, GSE123587.RData
    log: logs/seurat.log
    jobid: 2
    benchmark: benchmarks/analysis.txt

 singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged  --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg sh -c 'type conda'
 singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged  --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg sh -c 'conda --version'
 singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged  --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg sh -c 'conda info --json'
 singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged  --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg sh -c 'type conda'
 singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged  --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg sh -c 'conda --version'
 singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged  --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg sh -c 'conda info --json'
 singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged --bind /mnt/tank/scratch/mfiruleva/scn/config/7c2bd866:/mnt/tank/scratch/mfiruleva/scn/config/7c2bd866 --bind /mnt/tank/scratch/mfiruleva/scn/data/GSE123587:/mnt/tank/scratch/mfiruleva/scn/data/GSE123587 --bind /mnt/tank/scratch/mfiruleva/scn/stats/summary.csv:/mnt/tank/scratch/mfiruleva/scn/stats/summary.csv --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg bash -c 'source /opt/conda/bin/activate '\''/mnt/tank/scratch/mfiruleva/scn/config/65ef9760'\''; set -euo pipefail;  Rscript scripts/analysis.R 2> logs/seurat.log'
Activating singularity image /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg
Activating conda environment: /mnt/tank/scratch/mfiruleva/scn/config/65ef9760
Full Traceback (most recent call last):
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages/snakemake/executors.py", line 2078, in run_wrapper
    run(
  File "/mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged/rules/secondary_analysis.smk", line 29, in __rule_run_analysis
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages/snakemake/shell.py", line 176, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command ' singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged --bind /mnt/tank/scratch/mfiruleva/scn/config/7c2bd866:/mnt/tank/scratch/mfiruleva/scn/config/7c2bd866 --bind /mnt/tank/scratch/mfiruleva/scn/data/GSE123587:/mnt/tank/scratch/mfiruleva/scn/data/GSE123587 --bind /mnt/tank/scratch/mfiruleva/scn/stats/summary.csv:/mnt/tank/scratch/mfiruleva/scn/stats/summary.csv --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg bash -c 'source /opt/conda/bin/activate '\''/mnt/tank/scratch/mfiruleva/scn/config/65ef9760'\''; set -euo pipefail;  Rscript scripts/analysis.R 2> logs/seurat.log'' returned non-zero exit status 127.

[Fri Nov 13 14:08:08 2020]
Error in rule run_analysis:
    jobid: 2
    output: GSE123587.h5ad, GSE123587.RData
    log: logs/seurat.log (check log file(s) for error message)
    conda-env: /mnt/tank/scratch/mfiruleva/scn/config/65ef9760
    shell:
        Rscript scripts/analysis.R 2> logs/seurat.log
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages/snakemake/executors.py", line 2078, in run_wrapper
    run(
  File "/mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged/rules/secondary_analysis.smk", line 29, in __rule_run_analysis
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages/snakemake/shell.py", line 176, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command ' singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged --bind /mnt/tank/scratch/mfiruleva/scn/config/7c2bd866:/mnt/tank/scratch/mfiruleva/scn/config/7c2bd866 --bind /mnt/tank/scratch/mfiruleva/scn/data/GSE123587:/mnt/tank/scratch/mfiruleva/scn/data/GSE123587 --bind /mnt/tank/scratch/mfiruleva/scn/stats/summary.csv:/mnt/tank/scratch/mfiruleva/scn/stats/summary.csv --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg bash -c 'source /opt/conda/bin/activate '\''/mnt/tank/scratch/mfiruleva/scn/config/65ef9760'\''; set -euo pipefail;  Rscript scripts/analysis.R 2> logs/seurat.log'' returned non-zero exit status 127.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages/snakemake/executors.py", line 493, in _callback
    raise ex
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages/snakemake/executors.py", line 479, in cached_or_run
    run_func(*args)
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages/snakemake/executors.py", line 2160, in run_wrapper
    raise RuleException(
snakemake.exceptions.RuleException: CalledProcessError in line 12 of /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged/rules/secondary_analysis.smk:
Command ' singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged --bind /mnt/tank/scratch/mfiruleva/scn/config/7c2bd866:/mnt/tank/scratch/mfiruleva/scn/config/7c2bd866 --bind /mnt/tank/scratch/mfiruleva/scn/data/GSE123587:/mnt/tank/scratch/mfiruleva/scn/data/GSE123587 --bind /mnt/tank/scratch/mfiruleva/scn/stats/summary.csv:/mnt/tank/scratch/mfiruleva/scn/stats/summary.csv --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg bash -c 'source /opt/conda/bin/activate '\''/mnt/tank/scratch/mfiruleva/scn/config/65ef9760'\''; set -euo pipefail;  Rscript scripts/analysis.R 2> logs/seurat.log'' returned non-zero exit status 127.
  File "/mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged/rules/secondary_analysis.smk", line 12, in __rule_run_analysis

RuleException:
CalledProcessError in line 12 of /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged/rules/secondary_analysis.smk:
Command ' singularity exec --home /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged --bind /mnt/tank/scratch/mfiruleva/scn/config/7c2bd866:/mnt/tank/scratch/mfiruleva/scn/config/7c2bd866 --bind /mnt/tank/scratch/mfiruleva/scn/data/GSE123587:/mnt/tank/scratch/mfiruleva/scn/data/GSE123587 --bind /mnt/tank/scratch/mfiruleva/scn/stats/summary.csv:/mnt/tank/scratch/mfiruleva/scn/stats/summary.csv --bind /nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/site-packages:/mnt/snakemake /mnt/tank/scratch/mfiruleva/scn/config/e78b32002e592105c700c757fbe8a8e4.simg bash -c 'source /opt/conda/bin/activate '\''/mnt/tank/scratch/mfiruleva/scn/config/65ef9760'\''; set -euo pipefail;  Rscript scripts/analysis.R 2> logs/seurat.log'' returned non-zero exit status 127.
  File "/mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged/rules/secondary_analysis.smk", line 12, in __rule_run_analysis
  File "/nfs/home/mfiruleva/anaconda3/envs/scn_cor/lib/python3.8/concurrent/futures/thread.py", line 57, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /mnt/tank/scratch/mfiruleva/scn/data/GSE123587/merged/.snakemake/log/2020-11-13T140734.564386.snakemake.log
unlocking
removing lock
removing lock
removed all locks
